test/test.js

```diff
@@ -146,6 +146,17 @@ describe('ScssCommentParser', function () {
           value: 'font-weight: bold;\n  color: red;\n  .bar {\n    color: blue;\n  }'
         });
       });
+
+      it('should add line numbers if lineNumberFor provided', function(){
+        var lineNumberFor = function (index) { return index; }
+        var context = parser.contextParser(getContent('rule.test.scss'), lineNumberFor);
+        assert.deepEqual(context, {
+          type: 'css',
+          name: '.foo',
+          value: 'font-weight: bold;\n  color: red;\n  .bar {\n    color: blue;\n  }',
+          line: { start: 6, end: 74 }
```

Comment:
Yeah it's just the mock, I didn't want to write a real implementation of `lineNumberFor` just for testing purposes :-) You can see above that the mock just returns the index in the string without doing anything to it.